### ISSUE TITLE
Skip full build for documentation changes 

### ISSUE THREAD

**Issue by jakedsouza** [vmware/vic#4125]
**User Statement:**
As a vic engineer/vic documentation editor. I would like to skip most of the CI build except for linting the documentation change. 

**Details:**
Provide ability to skip code build and only run misspell and any other build steps related to docs. 
This could be a check from the pull-request title e.g [doc-change] which triggers a subset of the build. 

**Acceptance Criteria:**
Full ci build should be skippable for doc changes

**Comment by mhagen-vmware** [comment]
[skip ci] is supported already, that is what doc changes should be using, please re-open if I misunderstand this.

P.S. we currently support:
[skip ci]
[full ci]
[specific ci=<suite>]

**Comment by anchal-agrawal** [comment]
@mhagen-vmware `skip ci` skips the `misspell` target, which can cause future builds to fail if a doc change is merged with `skip ci`. To prevent this, the current practice is to not use `skip ci` for doc changes. Instead, we should come up with a solution that uses just the `misspell` target for doc changes (or something similar).

**Comment by mhagen-vmware** [comment]
good point, I like Jake's suggested [doc change].  We can implement that easily enough.
